Examples/ArcGISToolkitExamples/MeasureExample.swift

```diff
@@ -46,14 +46,7 @@ class MeasureExample: MapViewController{
         super.viewDidLayoutSubviews()
         
         // update content inset for mapview
-        let tbHeight = measureToolbar.frame.height
-        
-        if #available(iOS 11.0, *) {
-            mapView.contentInset = UIEdgeInsetsMake(0, 0, view.safeAreaInsets.bottom + tbHeight, 0)
-        }
-        else{
-            mapView.contentInset = UIEdgeInsetsMake(0, 0, tbHeight, 0)
-        }
+        mapView.contentInset = UIEdgeInsetsMake(0, 0, measureToolbar.frame.height, 0)
```

Comment:
I would suggest switching to using the `UIEdgeInset` initializer `init(top:left:bottom:right:)`:
```suggestion
        mapView.contentInset = UIEdgeInsets(top: 0, left: 0, bottom: measureToolbar.frame.height, right: 0)
```